R/run.R

```diff
@@ -20,16 +20,68 @@ run.default <- function(x, ...) {
 }
 
 
+#' @param publish Flag for whether to publish results to a Conbench server. See
```

Comment:
It might help to have a name + description for this function (or is that elsewhere already?)

Reply:
Yeah, that's on the generic. Currently only this method is really useful and all the docs are particular to it, but eventually it might be nice to write a method that would dispatch to `run_benchmark()` for a single benchmark.